populateDb/index.js

```diff
@@ -0,0 +1,36 @@
+const { DynamoDBClient, BatchWriteItemCommand } = require( "@aws-sdk/client-dynamodb");
+const ProductsSource = require("../productService/products.json");
+
+(async () => {
+  const client = new DynamoDBClient({ region: "eu-west-1" });
+  const batchCreateParams = {RequestItems: {}};
+  batchCreateParams.RequestItems.cloudX_products = ProductsSource.map(x => ({
```

Comment:
Better to pass DB table names as environment variables - it could be different on different environments